### DISCUSSION TITLE
Update game on Xbox Series S

### DISCUSSION THREAD

**Discussion by malvarenga123** [diasurgical/DevilutionX#5599] [Q&A] [answered]
Hi guys, I was able to install Devilution on my Xbox Series S using a link in Gamr13 discord server.
For some reason those links get deleted frequently, so I wanted to know if there's a way to use the files available here to update to future versions of DevilutionX without relying on third-party links.

BTW, I noticed two issues:
1) There's no way to run vanilla Diablo if the Hellfire MPQs are present. Selecting Diablo-only at the start is ignored
2) The game's folder does not have Gamr13 or Devilution in its name. It's all random characters (looks like Base64 encoded names). I could only find the right one because of the Diasurgical folder that gets created inside Localstate.

**Comment by AJenbo** [comment, marked as answer]
Afaik only if you set your Xbox to dev mode. What version did you get?

1. use the settings menu
2. it's probably UUID, not a form of encoding, talk to Microsoft why they think this is a good solution for resource managment

**Comment by malvarenga123** [comment]
>Afaik only if you set your Xbox to dev mode.

Oh I see. It's a paid feature right?

>What version did you get?

It shows up in the main screen as 1.4.1

>use the settings menu

In the settings it's set to Diablo, but I can still select every time I start the game and it's never Diablo, always Hellfire.

**Reply by AJenbo** [reply]
> Oh I see. It's a paid feature right?

I don't think so, only if you want to publish or something. But I don't have an Xbox so not sure.

> It shows up in the main screen as 1.4.1

Sounds like latest then

> In the settings it's set to Diablo, but I can still select every time I start the game and it's never Diablo, always Hellfire.

Sounds like it doesn't have permissions to write the settings to disk

**Reply by tsunamistate** [reply]
> Oh I see. It's a paid feature right?

Unfortunately :(

https://www.howtogeek.com/703443/how-to-put-your-xbox-series-x-or-s-into-developer-mode/

To activate Dev Mode on your Xbox, you have to register an individual app developer’s account with the [Microsoft Partner Center](https://click.linksynergy.com/deeplink?id=2QzUaswX1as&mid=24542&u1=htg/703443|xid:fr1670858337aaa&murl=https%3A%2F%2Fpartner.microsoft.com) ($19 in the U.S, but the cost varies in other regions).
